### DISCUSSION TITLE
Include nested

### DISCUSSION THREAD

**Discussion by ragokan** [prisma/prisma#5810] [Q&A]
Hello, I don't know how can I explain this well, so let me show it first.

![image](https://user-images.githubusercontent.com/63511519/108907828-ab0b8280-7633-11eb-9f27-081817b036f2.png)

As you can see, a comment has it's own comments.
When I do include all comments just like this:
![image](https://user-images.githubusercontent.com/63511519/108907896-c2e30680-7633-11eb-8461-c09f0a15849c.png)

I just get comments' comments but I can't get comments' comments' comments, how can I manage to do it without code repeat?

**Comment by ryands17** [comment]
@ragokan 👋 

Unfortunately specifying infinite levels of nesting is not possible at the moment, so you would need to explicitly specify the levels of nesting that you require:

```ts
let commentIncludeOptions: Prisma.CommentInclude = {
  comments: { include: { comments: { include: { comments: true } } } },
}
```

I would suggest creating a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for the same with the above use case so that we can look into this :)

**Reply by ragokan** [reply]
Thank you, I will surely do.

**Reply by wangyong220531** [reply]
It so troubling that specifying infinite levels of nesting is not possible, I'd like to define it as a bug in Prisma.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
